cluster/common/flux/variables.tf

```diff
@@ -4,6 +4,10 @@ variable "flux_repo_url" {
   default = "https://github.com/weaveworks/flux.git"
 }
 
+variable "gitops_path" {
+    type = "string"
+}
```

Comment:
I'd set this to a default of `/` so all environments don't have to specify a value.  This will break all our other environments.